--- pybind/mhm/CMakeLists.txt
+++ pybind/mhm/CMakeLists.txt
@@ -1,6 +1,6 @@
 enable_language(C)
 find_package(PythonExtensions REQUIRED)
-find_package(NumPy REQUIRED)
+find_package(Python REQUIRED COMPONENTS NumPy)
 find_package(F2PY REQUIRED)
 
 install(TARGETS mhm_lib LIBRARY DESTINATION .)
@@ -59,7 +59,7 @@ target_link_libraries(${f2py_module_name} mhm_lib)
 target_include_directories(${f2py_module_name}
   PUBLIC
   ${F2PY_INCLUDE_DIRS}
-  ${NumPy_INCLUDE_DIRS}
+  ${Python_NumPy_INCLUDE_DIRS}
   ${PYTHON_INCLUDE_DIRS}
 )
 
